packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -25,7 +27,8 @@ export const SafeLocalStorageKeys = {
   CONNECTED_SOCIAL: '@appkit/connected_social',
   CONNECTED_SOCIAL_USERNAME: '@appkit/connected_social_username',
   RECENT_WALLETS: '@appkit/recent_wallets',
-  DEEPLINK_CHOICE: 'WALLETCONNECT_DEEPLINK_CHOICE'
```

Comment:
We need to keep `DEEPLINK_CHOICE` as `WALLETCONNECT_DEEPLINK_CHOICE` since universal provider package uses that value.